static/graph-layout-core.ts

```diff
@@ -655,12 +655,14 @@ export class GraphLayoutCore {
         // Throw everything away and do it all again, but smarter
         for (const edgeColumn of this.edgeColumns) {
             for (const intervalTree of edgeColumn.intervals) {
-                intervalTree.root = null as unknown as Node<EdgeSegment>;
+                // Note: Incorrect types in the interval tree library
```

Comment:
Is there a repo we can file an issue against (and x-ref it here?)

Reply:
There is, I will go ahead and do so (https://github.com/alexbol99/flatten-interval-tree)

Reply:
https://github.com/alexbol99/flatten-interval-tree/pull/42 was merged so I will make a followup pr